test/index.js

```diff
@@ -147,6 +152,14 @@ var wrap = function(before) {
   };
 };
 
+var subAccum = function(a) {
```

Comment:
I don't mind this being defined separately, but I would like to see it defined closer to its call site (somewhere in the `test('traverse', ...)` block).